src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -331,6 +340,14 @@ const Notebooks = _.flow(
       notebooks && h(PageBox, [
         div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' } }, [
           div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Notebooks']),
+          label({ style: { marginLeft: 'auto', marginRight: '0.75rem' } }, 'Sort By: '),
```

Comment:
You should link this label to the select by id (I think @panentheos made a utility), or else just put it in a span or something.

Reply:
If you want to do this, look at usages of `IdContainer`

Reply:
Changed to a div